bitnami/postgresql-ha/values.yaml

```
@@ -863,6 +863,12 @@ pgpool:
   ## ref: https://github.com/bitnami/bitnami-docker-pgpool#configuration
   ##
   numInitChildren: ""
+  ## @param pgpool.reservedConnections When this parameter is set to 1 or greater, incoming connections from clients
+  ## are not accepted with error message "Sorry, too many clients already", rather than blocked if the number of current
+  ## connections from clients is more than (num_init_children - reserved_connections).
+  ## ref: https://github.com/bitnami/bitnami-docker-pgpool#configuration
+  reservedConnections: 1
+
```

This metadata is used to automatically generate the table present in the README using https://github.com/bitnami-labs/readme-generator-for-helm

As the description is very long, can you please split it into a simple first line and then add more clarifications or links if you want in the next lines?
```suggestion
  ## @param pgpool.reservedConnections When this parameter is set to 1 or greater, incoming connections from clients
  ## are not accepted with error message "Sorry, too many clients already", rather than blocked if the number of current
  ## connections from clients is more than (num_init_children - reserved_connections).
  ## ref: https://github.com/bitnami/bitnami-docker-pgpool#configuration
  ##
  reservedConnections: 1
```
Apart from that I just provide a suggestion for removing the blank line and adding the `##` before the value itself as in the rest of parameters

---

Understood. I assumed the REAMDE generation was taken care of by CI or something. I used that description as numInitChildren had taken its description verbatim from the pgpool docs site, so I assumed that was the policy. I kept the full description below, with a shorter one on the first line to be shown in the readme. Let me know if this is what you were looking for.

---

Thanks, now it looks much better.
Yes, there is some automation in place to generate this table, although at this very moment we are iterating over the process to improve it since it is a recent addition.